test/integration/features/story_health.feature

```diff
@@ -51,8 +51,18 @@ Feature:
         When executing "oc rollout status dc/httpd-ex" succeeds
         Then stdout should contain "successfully rolled out"
         And executing "oc expose svc/httpd-ex" succeeds
-        And with up to "2" retries with wait period of "60s" http response from "http://httpd-ex-testproj.apps-crc.testing" should have status code "200"
+        And with up to "2" retries with wait period of "60s" http response from "http://httpd-ex-testproj.apps-crc.testing" has status code "200"
 
+    @darwin @linux
+    Scenario: Stop and start CRC, then check app still runs
+        Given with up to "2" retries with wait period of "60s" http response from "http://httpd-ex-testproj.apps-crc.testing" has status code "200"
+        When executing "crc stop" succeeds
+        Then with up to "4" retries with wait period of "2m" command "crc status" output should contain "Stopped"
+        When starting CRC with default bundle and default hypervisor succeeds
+        Then with up to "4" retries with wait period of "2m" command "crc status" output should match ".*Running \(v\d+\.\d+\.\d*\).*"
+        And with up to "2" retries with wait period of "60s" http response from "http://httpd-ex-testproj.apps-crc.testing" has status code "200"
```

Comment:
:+1:

Reply:
I'd tend to split everything except this hunk in a separate commit.